c172p-set.xml

```diff
@@ -881,9 +881,6 @@ http://forum.flightgear.org/viewtopic.php?f=4&t=25157
 
         <vne>158</vne>
 
-        <!-- 2550*3.8 -->
-        <max-lift-force>9690</max-lift-force>
```

Comment:
This one can indeed be removed. It was used in older revisions of `Systems/c172p-damage.xml`.